lightning/checkpoints/file_checkpoints.proto

```diff
@@ -65,4 +65,5 @@ message ChunkCheckpointModel {
     int32 type = 14;
     int32 compression = 15;
     string sort_key = 16;
+    int64 size = 17;
```

Comment:
maybe another name such as 'filesize' could make code generator not add trailing `_`

Reply:
why is `size` not equals to `end_offset - offset`?

Reply:
size is the file size, not the chunk size.

Reply:
ok then please do rename it to "filesize".

Reply:
And for parquet, The offset/end_offset is line number, not actual file offset since parquet is column based